front-end/src/components/AnalyzersAndSeries.js

```diff
@@ -2,9 +2,8 @@ import '../styles/modalStyles.css';
 import Help from './Help';
 import analyzerData from './analyzerData';
 import '../styles/analyzersAndSeriesStyles.css';
-import React, { useState } from 'react';
-
-const AnalyzersAndSeries = ({ dimensions, columns, setDisplayPage, setShowModal, handleSubmit }) => {
+import { useState } from 'react';
+const AnalyzersAndSeries = ({ setSuccessMessage,success,dimensions, columns, setDisplayPage, setShowModal, handleSubmit }) => {
```

Comment:
In the future, please just add it at the end. Easier to follow

Reply:
Also, use spaces! Format correctly pls

Reply:
https://www.youtube.com/watch?v=unccXlME1X4